nengo/params.py

```diff
@@ -27,6 +27,13 @@ def is_param(obj):
     return isinstance(obj, Parameter)
 
 
+def params(obj):
```

Comment:
I would suggest `get_params` for the name of this function. `params` is too easily confusable with the module.